src/main/resources/org/keycloak/gh/bot/bug-action-messages.properties

```diff
@@ -0,0 +1,35 @@
+action-QUESTION=\
+  Thanks for reporting this issue, but this looks like a request for help and not a bug.\n\
+  \n\
+  For help and questions around Keycloak please see [keycloak.org/community](https://www.keycloak.org/community).
+
+action-MISSING_INFO=\
+  Thanks for reporting this issue, but there is insufficient information or lack of steps to reproduce.\n\
+  \n\
+  Please provide additional details, otherwise the issue will be automatically closed within 14 days.
+
+action-OLD_RELEASE=\
+  Thanks for reporting this issue, but as this is reported against an older and unsupported release we are not able \
+  to evaluate the issue. Please verify with the latest release and update the issue if the issue still exists.\n\
```

Comment:
```suggestion
  to evaluate the issue. Please verify with the [nightly build](https://github.com/keycloak/keycloak/releases/tag/nightly) if possible, or [latest release](https://www.keycloak.org/downloads) at a minimum.  Then update the issue if the issue still exists.\n\
```

Reply:
Update the message to reflect your suggestion